pom.xml

```diff
@@ -12,7 +12,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <spring-boot.version>1.5.4.RELEASE</spring-boot.version>
+        <spring-boot.version>2.0.0.M4</spring-boot.version>
```

Comment:
Travis complains about this:

```
[ERROR] Failed to execute goal on project consul-java-client: Could not resolve dependencies for project com.rebuy:consul-java-client:jar:1.5.1-SNAPSHOT: Failure to find org.springframework.boot:spring-boot:jar:2.0.0.M4 in http://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
```

Reply:
i'm still waiting for GA before this one is going to be released